examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,246 @@ Documentation for other releases can be found at
+WordPress image includes an Apache server).
+
+Demonstrated Kubernetes Concepts:
+
+* [Persistent Volumes](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/persistent-volumes.md) to
+  define persistent disks (disk lifecycle not tied to the Pods).
+* [Services](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/services.md) to enable Pods to
+  locate one another.
+* [External Load Balancers](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/services.md#type-loadbalancer)
+  to expose Services externally.
+* [Deployments](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/deployments.md) to ensure Pods
+  stay up and running.
+* [Secrets](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/secrets.md) to store sensitive
+  passwords.
+
+## tl;dr Quickstart
+
+Put your desired mysql password in a file called `password.txt` with no
+trailing newline.
+
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
+kubectl create secret generic mysql-pass --from-file=password.txt
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/mysql-deployment.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/wordpress-deployment.yaml
 ```
 
-Please see the [GCE getting started guide](../../docs/getting-started-guides/gce.md) for full details and other options for starting a cluster.
-
-## Create two persistent disks
-
-For this WordPress installation, we're going to configure our Kubernetes [pods](../../docs/user-guide/pods.md) to use [persistent disks](https://cloud.google.com/compute/docs/disks). This means that we can preserve installation state across pod shutdown and re-startup.
-
-You will need to create the disks in the same [GCE zone](https://cloud.google.com/compute/docs/zones) as the Kubernetes cluster. The default setup script will create the cluster in the `us-central1-b` zone, as seen in the [config-default.sh](../../cluster/gce/config-default.sh) file. Replace `$ZONE` below with the appropriate zone.
-
-We will create two disks: one for the mysql pod, and one for the wordpress pod. In this example, we create 20GB disks, which will be sufficient for this demo. Feel free to change the size to align with your needs, as wordpress requirements can vary. Also, keep in mind that [disk performance scales with size](https://cloud.google.com/compute/docs/disks/#comparison_of_disk_types).
-
-First create the mysql disk.
-
-```sh
-gcloud compute disks create --size=20GB --zone=$ZONE mysql-disk
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+## Table of Contents
+
```

Comment:
Is this the correct place for the table of contents?  Should it go before the tl;dr?  Before the what you will learn?  We should make the structuring consistent across examples.